DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj

```diff
@@ -1,14 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <ProjectGuid>12B1A410-DE8B-4F45-97E1-E0B502FDECC7</ProjectGuid>
-    <IncludeSymbols>false</IncludeSymbols>
+    <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <AssemblyName>DFC.TestAutomation.UI</AssemblyName>
     <OutputType>Library</OutputType>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
This caused issues on the pipeline a few weeks back and we had to set this to false. I think however, this was only a problem because we wanted to use the nuspec and this was essentially superseding the nuspec. Just an FYI - no action needed.

Reply:
I'm not sure if the pipeline nuget pack requires this, but other nugets I've inspected use it.  Will investigate if issues arise